### PR TITLE
workers added

### PR DESCRIPTION
**Pull request by JoeyL626**
and the other create end points changed back to add like the models say to use

### CODE REVIEW

File: server/routes/api/v1/pod/create.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ module.exports = function (express) {
 
     // Example of data in JSON format
     // pod.add({current_weight: 124, max_weight: 900, last_maintain: 1200}
-    pod.create({current_weight: clientPodPost.current_weight, max_weight: clientPodPost.max_weight, last_maintain: clientPodPost.last_maintain},
+    pod.add({current_weight: clientPodPost.current_weight, max_weight: clientPodPost.max_weight, last_maintain: clientPodPost.last_maintain},
```
Comment (Vercjames) [inline server/routes/api/v1/pod/create.js]: pod.create should be the notation not .add. Please pull master to make sure that your model.js reflect an create function return not an add

File: server/routes/api/v1/unit/create.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ module.exports = function (express) {
 
     // Example of data in JSON format
     // unit.add({availability_qty: 3, trigger_qty:4, replenish_qty:5, description: "pink", weight_lbs: 4}
-    unit.create({availability_qty: clientUnitPost.availability_qty, trigger_qty: clientUnitPost.trigger_qty, replenish_qty: clientUnitPost.replenish_qty, description: clientUnitPost.description, weight_lbs: clientUnitPost.weight_lbs},
+    unit.add({availability_qty: clientUnitPost.availability_qty, trigger_qty: clientUnitPost.trigger_qty, replenish_qty: clientUnitPost.replenish_qty, description: clientUnitPost.description, weight_lbs: clientUnitPost.weight_lbs},
```
Comment (Vercjames) [inline server/routes/api/v1/unit/create.js]: unit.create should be the notation not .add. Please pull master to make sure that your model.js reflect an create function return not an add

File: server/routes/api/v1/pod/create.js (hunk 1 above)
Comment (JoeyL626) [inline server/routes/api/v1/pod/create.js]: according to the models its add i did pull master and then changed them according to what the models say i can change it back though

File: server/routes/api/v1/unit/create.js (hunk 2 above)
Comment (JoeyL626) [inline server/routes/api/v1/unit/create.js]: according to the models its add i did pull master and then changed them according to what the models say i can change it back though

File: server/routes/api/v1/pod/create.js (hunk 1 above)
Comment (Vercjames) [inline server/routes/api/v1/pod/create.js]: Then your models are out dated

Comment (JoeyL626) [inline server/routes/api/v1/pod/create.js]: i just looked at the models from master on github but i will change them back and pull it again

Comment (JoeyL626) [inline server/routes/api/v1/pod/create.js]: should all the create end points be create?

Comment (Vercjames) [inline server/routes/api/v1/pod/create.js]: I'd delete your folder and redo the git settup. something is wrong locally. earlier you submitted 34 file changes and everything was screwy.

Comment (JoeyL626) [inline server/routes/api/v1/pod/create.js]: ignore this branch i made a new one and redid the git setup so it should be correct if its not let me know

**Pull request closed without merging**